package.json

```diff
@@ -12,13 +15,42 @@
     "url": "https://github.com/Krombik/html2react/issues"
   },
   "homepage": "https://github.com/Krombik/html2react#readme",
-  "type": "module",
-  "source": "src/index.ts",
   "engines": {
     "node": ">=10"
   },
+  "type": "module",
+  "source": "src/index.ts",
+  "main": "./build/index.cjs",
+  "module": "./build/index.js",
+  "types": "./build/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./build/index.js",
+      "default": "./build/index.cjs",
+      "types": "./build/index.d.ts"
+    },
+    "./HTML2React": {
+      "import": "./build/HTML2React/index.js",
+      "default": "./build/HTML2React/index.cjs",
+      "types": "./build/HTML2React/index.d.ts"
+    },
+    "./styleConverter": {
+      "import": "./build/styleConverter/index.js",
+      "default": "./build/styleConverter/index.cjs",
+      "types": "./build/styleConverter/index.d.ts"
+    }
+  },
+  "files": [
```

Comment:
This replicates the copying you were doing before, just specifying what is relevant to be bundled. `src` is included since the package.json `source` field is set.